lib/client-assets.php

```diff
@@ -146,10 +157,11 @@ function gutenberg_register_scripts_and_styles() {
 	);
 	wp_register_script(
 		'wp-blocks',
-		gutenberg_url( 'blocks/build/index.js' ),
+		gutenberg_url( 'blocks/build/index.js', GUTENBERG_WEBPACK_HMR  ),
```

Comment:
It's a bit odd to me that we pass the `GUTENBERG_WEBPACK_HMR` constant value in as an argument, as though we'd want to treat the function as generic and unaware of the constants, but then proceed to use `GUTENBERG_WEBPACK_DEV_SERVER` within the function implementation. I'd suggest going consistent one way or the other (either passing both as arguments, or referencing both within the function).